tests/test_cursor.py

```diff
@@ -105,7 +104,6 @@ def test_mogrify_decimal_explodes(self):
         self.assertEqual(b'SELECT 10.3;',
             cur.mogrify("SELECT %s;", (Decimal("10.3"),)))
 
-    @skip_if_no_getrefcount
```

Comment:
Which PyPy versions do you wish to support?

I expect the most recent versions, PyPy2.7 v5.9.0 and PyPy3.5 v5.9.0, have `sys.getrefcount`.

Reply:
It was needed time ago, when psycopg was ported. I haven't really followed PyPI development.

I expect them not having `getrefcount` because they are not based on refcount. In doubt, please leave it there.

Reply:
Is PyPy even a supported platform? It isn't included in the Travis CI or tox test matrix. It also isn't listed in the trove classifiers.

Reply:
It is not supported by this implementations, but others do:

- https://github.com/chtd/psycopg2cffi/
- https://github.com/mvantellingen/psycopg2-ctypes

They are sort of semi-official, but our condition for them to be called "psycopg" is that they pass psycopg test suite. This was one of the tweaks that `-ctypes` needed and it was imported back in the upstream test suite.

Reply:
Interesting. Thanks!